### PR TITLE
Add dev blog for nft tutorial

### PR DESCRIPTION
**Pull request by amarantha-k**

### CODE REVIEW

**Comment by AKCodez** [comment]
Looks good other than the fact that is missing an SEO description

File: blog/2024/how-to-mint-nfts.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+date: 2024-03-08
+labels:
+    - Development
+targets: [devblog]
+---
+# How to Mint an NFT on the XRP Ledger: A Step-by-Step Guide
+
+The first in a broader series dedicated to making your XRP Ledger building journey an altogether smoother experience is here! In this piece we cover how you can begin to shape your very own NFT collection on the XRP Ledger today. 
+
+Navigating the exciting realm of [Non-fungible Tokens (NFTs)](https://xrpl.org/docs/concepts/tokens/nfts/) on the XRP Ledger can seem daunting at first. However, with the right tools and a bit of JavaScript, minting, managing, and burning NFTs becomes a straightforward process. This blog demystifies the process, providing a clear guide on how to create, view, and destroy NFTs on this innovative platform.
```
Comment (justinr1234) [inline blog/2024/how-to-mint-nfts.md]: Are there plans for a python version?

File: blog/2024/how-to-mint-nfts.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+<!-- BREAK -->
+
+
+## How Much Does it Cost to Mint an NFT?
+
+Minting, holding, and offering NFTs for sale require XRP held in reserve. The reserve charges can add up quickly. Understanding the reserve requirements can help you choose the best approach for your business case.
```
Comment (justinr1234) [inline blog/2024/how-to-mint-nfts.md]: `requires` instead of `require`.

I’d combine and say

The reserve charge can add up quickly, so understanding..

**Review by justinr1234 — APPROVED** [review]
Looks good nicely done! Only minor comments which can be ignored.

**Comment by amarantha-k** [comment]
Good catch, @AKCodez! Just updated with the meta data.

File: blog/2024/how-to-mint-nfts.md (hunk 1 above)
Comment (amarantha-k) [reply blog/2024/how-to-mint-nfts.md]: Good point, thanks for catching this. 
The Python version of the tutorial already exists: https://xrpl.org/docs/tutorials/python/modular-tutorials/nfts/mint-and-burn-nfts/, so updated the blog post to include a link to Python too.

**Pull request merged**